interface/forms/ros/C_FormROS.class.php

```diff
@@ -1,30 +1,47 @@
 <?php
-
+/** Copyright (C) 2016 Sherwin Gaddis <sherwingaddis@gmail.com>
+ *
+ * LICENSE: This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://opensource.org/licenses/gpl-license.php>;.
+ *
+ * @package OpenEMR
+ * Sherwin Gaddis <sherwingaddis@gmail.com>
+ * @link    http://www.open-emr.org
+ */
+ 
 require_once ($GLOBALS['fileroot'] . "/library/classes/Controller.class.php");
 require_once ($GLOBALS['fileroot'] . "/library/forms.inc");
 require_once("FormROS.class.php");
 
 class C_FormROS extends Controller {
 
-	var $template_dir;
+	public $template_dir;
```

Comment:
I changed them because of the course on OOP that I am in. Everything we write for our classes is labeled public private or protected. And I was told that var was the old way. So, I was going to embark on updating how the classes are referenced. Since I saw it in the new code for the fee sheet. I thought it would be a two for one while updating the look of the ROS that I was asked to do. I thought I would sneak the changes in those classes in at this time. 
Even though at this time it makes no difference in the operation of the class. One of the other things that I learned was auto instantiation. That could cut the size of the FormROS in half or better if I can get it working. I'm just trying find something useful to do to be a part of the bigger picture.